### PR TITLE
Feature/#137 jax example

### LINKED ISSUE #137: Add JAX example

### PR DESCRIPTION
**Pull request by mhuguesaws**
*Issue #, if available:*
Close #137 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 2.ami_and_containers/containers/jax/jax_paxml.Dockerfile
Comment (perifaws) [inline 2.ami_and_containers/containers/jax/jax_paxml.Dockerfile:1]: Nitpicking here but should we prepend with a digit like for other files

Comment (mhuguesaws) [reply 2.ami_and_containers/containers/jax/jax_paxml.Dockerfile:1]: There is no reason to label thing with numbers
1. there are not steps.
2. will be impossible to maintain since we need to move folder numbering every time.

Comment (perifaws) [reply 2.ami_and_containers/containers/jax/jax_paxml.Dockerfile:1]: Makes sense, let's merge then... approved

**Review by perifaws — APPROVED** [review]

**Comment by mhuguesaws** [comment]
Not sure we should have the jax.sbatch in the container folder or the test_cases folder. It's getting confusing.

**Comment by perifaws** [comment]
Test cases of launched outside of the container.

On Feb 14, 2024, at 6:51 PM, mhuguesaws ***@***.***> wrote:

﻿

Not sure we should have the jax.sbatch in the container folder or the test_cases folder. It's getting confusing.
